marimbabot_planning/README.md

```diff
@@ -0,0 +1,23 @@
+# TAMS Master Project 2022/2023 - motion documentation
+
+In order to run the whole project on the real robot, launch the bringup package that brings up the launch file for each package:
+
+```bash
+roslaunch marimbabot_bringup marimbabot.launch
```

Comment:
When I run this launch,  will the gripper release the mallet?

Reply:
May we add such information in the README as well, please?